Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/wehny.yml

```
@@ -75,10 +75,12 @@
       groups:
         Brute: 50
   - type: Tackle
-    threshold: 5
-    stun: 8
+    threshold: 4
+    stun: 15
```

chief this is a stun longer than the queen's for only one hit more on a much faster attack rate
```suggestion
    threshold: 4
    stun: 15
```